src/sagas/index.js

```diff
@@ -708,6 +712,9 @@ function* startVm (action) {
   yield startProgress({ vmId: action.payload.vmId, name: 'start' })
   const result = yield callExternalAction('start', Api.start, action)
   // TODO: check status at refresh --> conditional refresh wait_for_launch
+  if (result.status === 'complete') {
+    yield put(addUserMessage({ message: 'VM Start Request Received', type: 'success' }))
```

Comment:
The text needed to be externalized (moved to `messages.js`).

I suggest naming the message key `toastVmStart`.